typing_extensions/pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "flit_core.buildapi"
 # Project metadata
 [project]
 name = "typing_extensions"
-version = "4.0.1"
+version = "4.1.0"
```

Comment:
Let's leave this out, we'll prepare a release separately. However, it would be good to add to the changelog.

Reply:
Not sure how to add this to the changelog. Do I just add a line above: # Release 4.0.1 (November 30, 2021)
Should I make a section for "Release 4.x.x" instead?

Reply:
Something like https://github.com/python/typing/pull/963/files#diff-ad6d05e61a5c54be368726cc13431904e0e23882704fe4def8605c311c92bcbc